### PR TITLE
Script Wrote for adding chromium

### PR DESCRIPTION
**Pull request by meets2tarun**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

Work on the issue https://github.com/fossasia/meilix-generator/issues/121

### CODE REVIEW

**Comment by xeon-zolt** [comment]
will chromium work on meilix ? 64 bit support only !!

**Comment by meets2tarun** [comment]
@xeon-zolt 
Yes, there is chromium exist for 32 bit also.

File: app.py [hunk 1]
```diff
@@ -36,6 +36,7 @@ def index():
         email = request.form['email']
         TRAVIS_TAG = request.form['TRAVIS_TAG']
         event_url = request.form['event_url']
+        GENERATOR_ = request.form['GENERATOR_']
```
Comment (niccokunzmann) [inline app.py]: Cool.
To get all variables, you should be able to do something like

    variables = {}
    for name, value in request.form.items():
        if name.startswith("GENERATOR_"):
            variables[name] = value

File: scripts/chromium-package.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+if echo "$GENERATOR_package_chromium" | grep -q chromium; then 
+sudo apt-get install -q -y chromium; fi
```
Comment (niccokunzmann) [inline scripts/chromium-package.sh]: Yep :+1:

File: templates/index.html [hunk 3]
```diff
@@ -499,7 +499,7 @@
 <tbody>
   <tr>
     <td>Chromium</td>
-    <td><input type="checkbox"></td>
+    <td><input name="GENERATOR_package_chromium" type="checkbox" value="chromium" id="chromium"></td>
```
Comment (niccokunzmann) [inline templates/index.html]: I am excited to see this in action one day.

**Pull request merged**